general-practices.md

```diff
@@ -0,0 +1,24 @@
+# JavaScript
```

Comment:
Does this JavaScript heading and intro make sense here?

Reply:
Doh.